### ISSUE TITLE
"node generate.js" results in "TypeError: Cannot convert undefined or null to object"

### ISSUE THREAD

**Issue by brentzundel** [w3c/vc-test-suite#45]
When running the shell command given on step 8, I received the following error:
```
brent@Lappy2000:~/dev/w3c/vc-test-suite$ cd implementations/ && node generate.js
/home/brent/dev/w3c/vc-test-suite/implementations/generate.js:25
      (Object.keys(test.err).length === 0) ? 'success' : 'failure';
              ^

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at results.tests.forEach (/home/brent/dev/w3c/vc-test-suite/implementations/generate.js:25:15)
    at Array.forEach (<anonymous>)
    at files.forEach (/home/brent/dev/w3c/vc-test-suite/implementations/generate.js:23:17)
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/brent/dev/w3c/vc-test-suite/implementations/generate.js:16:7)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
```

**Comment by brentzundel** [comment]
Dmitri, let me know what next steps could be to troubleshoot this.

**Comment by TallTed** [comment]
Retitling this issue would be helpful. Perhaps `"node generate.js" results in "TypeError: Cannot convert undefined or null to object"`

**Comment by dmitrizagidulin** [comment]
Fixed in PR #57.
